lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -180,4 +209,44 @@ module.exports = {
     }
     return event.alb.multiValueHeaders;
   },
+
+  validateEventAuthorizer(event, authorizers, functionName) {
```

Comment:
As it's about multiple authorizers, let's rename it to `validateEventAuthorizers`